test/glific_web/schema/ticket_test.exs

```diff
@@ -237,4 +237,21 @@ defmodule GlificWeb.Schema.TicketTest do
 
     assert ticket.message_number == message_number
   end
+
+  test "close multiple tickets and test possible scenarios and errors", %{manager: user} do
+    TicketsFixtures.ticket_fixture()
+
+    {:ok, ticket} =
+      Repo.fetch_by(Ticket, %{topic: "some topic", organization_id: user.organization_id})
+
+    params = %{
+      "topic" => [ticket.topic],
+      "status" => "closed"
+    }
+
+    {:ok, result_map} = Tickets.update_ticket_status_based_on_topic(params)
+
+    message = result_map |> Map.get(:message)
+    assert message == "Updated successfully"
+  end
```

Comment:
this is not a schema level test but a function test.

Schema tests are added here(here graphql query is replicated)
`test/glific_web/schema/ticket_test.exs`

Function test are added here(functions are called and output is matched)
`test/glific/tickets_test.exs`